.github/actions/sign-attestation/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Sign Attestation'
+description: 'Signs an in-toto attestation and returns a DSSE signed envelope in a trust bundle at the file attestation.jsonl'
+inputs:
+  attestation:
+    description: 'An in-toto attestation to sign'
+    required: true
+  payload_type:
```

Comment:
will this be a public Action or an internal one?
Shall we start with making it internal?

nit: Our Action typically follow `some-name` instead of `some_name`